agent/bench-scripts/postprocess/uperf-server-launcher

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "This is the uperf server launcher"
+echo "args: $@"
+
+port=`echo "20000 + 3 * $pbench_count" | bc`
+#existing_pid=`
+echo "/usr/local/bin/uperf -v -s -P $port >server-stdout.txt 2>server-stderr.txt" >server.cmd
+echo "exit $?" >>server.cmd
+chmod +x server.cmd
+./server.cmd & echo "$!" >server.pid
```

Comment:
The point of the current method is to allow someone to re-run server.cmd if they need to debug this.  We do this a lot in pbench now, and I was continuing that method.  If we don't want to continue that method, we can do what you describe.  The capture of the server pid is actually no longer necessary anyway, and we can remove that.

Reply:
I would prefer to not get rid of the ability to easily/quickly relaunch the commands when doing deep dive analysis/debug.

Reply:
I can understand the need to capture the command that is run, but since the command is only different here by `$port` it seems it is a bit of an overkill just to rerun the command given all the mechanism involved to do so (create a file, make sure it exits with a status value (which it would by default I believe), and then make it executable, and then run it).  Validating the arguments and then executing the command with an `exec` seems easy enough.

Regardless, it would be great to make sure we check the arguments properly.